bigtable/client/internal/readrowsparser_acceptance_tests.inc

```diff
@@ -454,7 +454,7 @@ TEST_F(AcceptanceTest, SplitCellBareCommit) {
           qualifier: < value: "C">
           timestamp_micros: 100
           value: "v"
-          value_size: 10
```

Comment:
What are all the numeric changes? Are they necessary? For testing? To force test re-runs?

Reply:
Good find... I regenerated the file using the [script](https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/master/bigtable/tools/convert_acceptance_tests.py), which downloads this [file](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/commits/master/bigtable-client-core-parent/bigtable-client-core/src/test/resources/com/google/cloud/bigtable/grpc/scanner/read-rows-acceptance-test.json) as its source data.  And evidently the file changed in the last couple of days.

Reply:
I should have added: that means I do not know what the changes are for, I am sure there is a good reason for them, but as long as the tests exercise the code and they pass, I am Okay with whatever values we want there.

Reply:
Welll, in that case...

![image](https://user-images.githubusercontent.com/4074659/36521028-8826fa52-1762-11e8-96b1-306ddc644782.png)